tracing/src/db.rs

```diff
@@ -0,0 +1,40 @@
+use crate::primitives::SpanChunk;
+use bson::doc;
+use mongodb::{Collection, IndexModel};
+
+/// Simple wrapper on top of mongodb.
+#[derive(Clone)]
+pub struct Database {
+    pub db: mongodb::Database,
+}
+
+impl Database {
+    /// The collection where we dump all the traces received by the collector.
+    pub fn span_chunks(&self) -> Collection<SpanChunk> {
+        self.db.collection("SpanChunk")
```

Comment:
does "Chunk" refer to our block chunk or a db concept here? if the former, can we use another name, as the trace will not be only chunks in some cases.

Reply:
It's just a db concept meaning a chunk of spans.